### PR TITLE
Include content

### PR DESCRIPTION
**Pull request by mikeyrayvon**
- includes page content on index with query
- lists page titles as menu items with query
- adds basic logic for hide/show page content

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -28,12 +28,33 @@
...
-      <li class="header-menu-item header-menu-active"><a href="#">Open Innovation Group</a></li>
-      <li class="header-menu-item"><a href="#">Selected Clients</a></li>
-      <li class="header-menu-item"><a href="#">ConnexI/O</a></li>
-      <li class="header-menu-item"><a href="#">Contact</a></li>
+<?php
+	while ( $query->have_posts() ) {
+		$query->the_post();
+?>
+
+      <li class="header-menu-item"><a href="#<?php echo $post->post_name; ?>"><?php the_title(); ?></a></li>
```
Comment (varffs) [inline header.php]: the issue with this is the link is just the text no the width of the `<li>`. that can be fixed with css or we trigger the hash link with pure js from the `<li>` click

File: index.php [hunk 2]
```diff
@@ -3,46 +3,33 @@
...
-    <div class="container">
-      <div class="grid-row">
 
 <?php
-if( have_posts() ) {
-  while( have_posts() ) {
-    the_post();
-?>
+$args = array(
+	'post_type'      => array( 'page' ),
```
Comment (varffs) [inline index.php]: tab indentation?

File: src/js/main.js [hunk 3]
```diff
@@ -52,10 +52,15 @@ Site.Menu = {
...
     var _this = this;
+    var href = $(item).children('a').attr('href');
+    var hash = href.substr(1);
 
     $('.header-menu-active').removeClass('header-menu-active');
     $(item).addClass('header-menu-active');
-  }
+
+    $('.page-content').removeClass('page-content-active');
+    $('.page-content[data-slug="' + hash + '"]').addClass('page-content-active');
```
Comment (varffs) [inline src/js/main.js:62]: I'm cool with this approach and it works well in practice. But I think the client asked for a full scroll in the design comments so need to clarify that

**Review by varffs — APPROVED** [review]
Not going to merge unless you have replies or want to make any edits but I approve the button pressing

File: index.php (hunk 2 above)
Comment (mikeyrayvon) [reply index.php]: oops. its copypasta from generatewp :<

File: src/js/main.js (hunk 3 above)
Comment (mikeyrayvon) [reply src/js/main.js:62]: mmm can you point out where they asked for full scroll? maybe on for mobile? tho i anticipated for mobile we would just show all the `.page-content` divs.

in the designs its not full scroll. check out the .ai:

screen shot 2017-05-09 at 3 19 37 pm

File: header.php (hunk 1 above)
Comment (mikeyrayvon) [reply header.php]: the point of the `<a>` is for keyboard navigation. its an accessibility issue. you can't **tab** thru `<li>`s

Comment (mikeyrayvon) [reply header.php]: so the `<a>`s should be `block` i guess

Comment (mikeyrayvon) [reply header.php]: if i get your meaning

File: src/js/main.js (hunk 3 above)
Comment (varffs) [reply src/js/main.js:62]: it said something to the effect of scroll in a pdf comment, but I dont really understand which files are final so maybe it has changed.

**Pull request merged**